TestProjects/Playground/Playground.Forms.Uwp/EmptyVoidLogProvider.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MvvmCross.Platform.Logging;
+
+namespace Playground.Forms.Uwp
+{
+
+    public class EmptyVoidLog : IMvxLog
```

Comment:
Why is this class needed?

Reply:
I added it because I was not able to make the default logger working. It was throwing exceptions